### PR TITLE
Limits number of authz polices to one per service and adds more info.

### PR DESCRIPTION
**Pull request by tomek-US**
- removes the Install() RPC
- replaces policy ID with the name of the service the new policy will be applied to.
- adds Probe() RPC
- adds Get() RPC
- expands the README.md

### CODE REVIEW

File: authz/authz.proto [hunk 1]
```diff
@@ -204,11 +172,82 @@ message UploadRequest {
...
+  string server = 4;
 }
 
-message UploadResponse {}
+message UploadResponse {
+}
 
+// ProbeRequest contains a single user name, the gRPC service name and the gRPC
+// call being attempted.
+message ProbeRequest {
```
Comment (marcushines) [inline authz/authz.proto:190]: do we think probe in this context is implementable?

**Review by marcushines — COMMENTED** [review]
everything else LGTM

File: authz/README.md [hunk 2]
```diff
@@ -174,29 +174,152 @@ will be able to call any of the `gNSI.ssh` RPCs.
...
+Authorization Policy, namely:
+
+1. Starting the `gNSI.authz.Rotate()` streaming RPC.
+
+   As the result a streaming connection is created between the server (the
+   switch) and the client (the management application) that is used in the
+   following steps.
+
+   > **⚠ Warning**
+   > Only one `gNSI.authz.Rotate()` can be in progress.
```
Comment (marcushines) [inline authz/README.md:205]: the auth policy is "declarative" for all servers and services on the device
All policies must be defined as previous policies will be replaced during the rotate

Comment (tomek-US) [reply authz/README.md:205]: done.

**Review by marcushines — APPROVED** [review]

**Pull request merged**